catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/converter/JdbcColumnDefaultValueConverter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.jdbc.converter;
+
+import static com.datastrato.gravitino.rel.Column.DEFAULT_VALUE_NOT_SET;
+
+import com.datastrato.gravitino.rel.expressions.Expression;
+import com.datastrato.gravitino.rel.expressions.FunctionExpression;
+import com.datastrato.gravitino.rel.expressions.literals.Literal;
+import com.datastrato.gravitino.rel.expressions.literals.Literals;
+import com.datastrato.gravitino.rel.types.Type;
+import java.time.format.DateTimeFormatter;
+
+public abstract class JdbcColumnDefaultValueConverter {
+
+  protected static final String CURRENT_TIMESTAMP = "CURRENT_TIMESTAMP";
+  protected static final String NULL = "NULL";
+  protected static final DateTimeFormatter DATE_TIME_FORMATTER =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+  public String fromGravitino(Expression defaultValue) {
+    if (DEFAULT_VALUE_NOT_SET.equals(defaultValue)) {
+      return null;
+    }
+
+    if (defaultValue instanceof FunctionExpression) {
+      FunctionExpression functionExpression = (FunctionExpression) defaultValue;
+      if (functionExpression.functionName().equalsIgnoreCase(CURRENT_TIMESTAMP)) {
+        return CURRENT_TIMESTAMP;
+      } else {
+        return String.format("(%s)", functionExpression);
```

Comment:
besides current_timestamp, do we support other function?

Reply:
Yes ,see tests :
https://github.com/datastrato/gravitino/blob/fdc4224fd188357028ab299950bc1dcab39535c2/integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/CatalogMysqlIT.java#L414-L423